.github/ISSUE_TEMPLATE/1-bug-report.md

```diff
@@ -0,0 +1,101 @@
+---
+name: "Bug Report"
+about: Report a bug
+labels: bug, needs-triage
+
+---
+<!--
+Thanks for filing an issue!  Please provide as much of the information requested below as possible.
+
+To expedite issue processing please search open and closed issues before submitting a new one.
+Existing issues often contain information about workarounds, resolution, or progress updates.
+
+Even better, we'd love a pull request to resolve the issue!
+-->
+
+# Bug Report
+
+### Description
+
+<!-- A clear and concise description of the problem. -->
+
+### Schema
+
+<!--
+Is this issue related to a specific schema?  Has the schema already been contributed? Can the schema be contributed for reproduction?  
+
+Provide the content of the schema or a link to the contributed schema if possible.
+-->
+
+### Is this a regression?
+
+<!-- Did this behavior work in a previous version?  What version? -->
+
+### Steps to reproduce
+
+<!-- Detailed steps for how to reproduce the observed behavior. -->
+
+### Exception or error
+
+<!-- Is there a specific exception or error message being seen? -->
+
+### Environment
+
+<!-- Confirm the following environment information as is relevant. --> 
+
+**Python Version:**
+
+<pre><code>
+<!-- run `pip --version` and paste the output below. -->
+
+</code></pre>
+
+**pip Version:**
+
+<pre><code>
+<!-- run `python --version` and paste the output below. -->
```

Comment:
Looks like you have the syntax for getting pip version in the field for Python and vice versa

Reply:
🤦  sure did, fixed in the last commit.